controllers/profiles/object_creators.go

```diff
@@ -59,11 +58,13 @@ const (
 	healthStartedInitialDelaySeconds = 10
 )
 
-var defaultDevApplicationProperties = "quarkus.http.port=" + strconv.Itoa(defaultHTTPWorkflowPort) + "\n" +
+var defaultDevApplicationProperties = "quarkus.http.port=8080\n" +
```

Comment:
Please change the `defaultHTTPWorkflowPort` to string to avoid the conversion.

Reply:
`defaultHTTPWorkflowPort` is used as int here https://github.com/kiegroup/kogito-serverless-operator/pull/130/files#diff-f6579aaafc7e439d9ba8180773a600072a37fb226296cb1ea4bb4670d10a3174R104

and as intoStr https://github.com/kiegroup/kogito-serverless-operator/pull/130/files#diff-f6579aaafc7e439d9ba8180773a600072a37fb226296cb1ea4bb4670d10a3174R111

here https://github.com/kiegroup/kogito-serverless-operator/pull/130/files#diff-f6579aaafc7e439d9ba8180773a600072a37fb226296cb1ea4bb4670d10a3174R120

and here https://github.com/kiegroup/kogito-serverless-operator/pull/130/files#diff-f6579aaafc7e439d9ba8180773a600072a37fb226296cb1ea4bb4670d10a3174R129

I'll prepare three type